docs/documentation/parameterization.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+title: Syntax for parameterizing specific fields
+permalink: /documentation/parameterization-syntax
```

Comment:
Can we move this to the parameterizer transformer documentation? This will help keep all transformers as equal. We will keep adding the documentation for all transformers. It should be in customizations/purpose-built/parameterizer.md